js/app/improve-this-page.js

```diff
@@ -1,179 +1,121 @@
 $(document).ready(function () {
     var pageURL = window.location.href;
-    var feedbackOrigin = window.feedbackOrigin;
-    var useFeedbackAPI = document.querySelector("#feedback-api-enabled");
-    if (document.querySelector("#feedback-api-enabled") && 
-        document.querySelector("#feedback-api-enabled").value === "true" && 
-        document.querySelector("#feedback-api-url")) {
-        var feedbackURL = document.querySelector("#feedback-api-url").value;
-    } else {
-        var feedbackURL ="/feedback";
-    }
-
-    if (feedbackOrigin && feedbackOrigin.length > 0) {
-        feedbackURL = feedbackOrigin + feedbackURL;
-    }
-
+    var feedbackURL = document.querySelector("#feedback-api-url").value;
```

Comment:
Replace the `var` with `const` and prefix jQuery objects with a `$` 
```suggestion
    const $feedbackURL = document.querySelector("#feedback-api-url").value;
```